{"abstract":[{"text":"additional context information related to the error","type":"text"}],"kind":"symbol","sections":[],"identifier":{"url":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSResponseError\/context","interfaceLanguage":"swift"},"schemaVersion":{"patch":0,"major":0,"minor":3},"primaryContentSections":[{"declarations":[{"languages":["swift"],"platforms":["macOS"],"tokens":[{"text":"let","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"context"},{"kind":"text","text":": "},{"text":"AWSErrorContext","preciseIdentifier":"s:8SotoCore15AWSErrorContextV","kind":"typeIdentifier","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSErrorContext"},{"kind":"text","text":"?"}]}],"kind":"declarations"}],"hierarchy":{"paths":[["doc:\/\/codes.soto.soto-core\/documentation\/SotoCore","doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSResponseError"]]},"variants":[{"paths":["\/documentation\/sotocore\/awsresponseerror\/context"],"traits":[{"interfaceLanguage":"swift"}]}],"metadata":{"modules":[{"name":"SotoCore"}],"role":"symbol","title":"context","externalID":"s:8SotoCore16AWSResponseErrorV7contextAA15AWSErrorContextVSgvp","symbolKind":"property","roleHeading":"Instance Property","fragments":[{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"context","kind":"identifier"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"AWSErrorContext","preciseIdentifier":"s:8SotoCore15AWSErrorContextV"},{"text":"?","kind":"text"}]},"references":{"doc://codes.soto.soto-core/documentation/SotoCore/AWSResponseError/context":{"abstract":[{"text":"additional context information related to the error","type":"text"}],"title":"context","role":"symbol","kind":"symbol","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSResponseError\/context","type":"topic","url":"\/documentation\/sotocore\/awsresponseerror\/context","fragments":[{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"text":"context","kind":"identifier"},{"text":": ","kind":"text"},{"text":"AWSErrorContext","kind":"typeIdentifier","preciseIdentifier":"s:8SotoCore15AWSErrorContextV"},{"text":"?","kind":"text"}]},"doc://codes.soto.soto-core/documentation/SotoCore/AWSResponseError":{"role":"symbol","title":"AWSResponseError","abstract":[{"type":"text","text":"Response Error type returned by Soto if the error code is unrecognised"}],"type":"topic","url":"\/documentation\/sotocore\/awsresponseerror","navigatorTitle":[{"text":"AWSResponseError","kind":"identifier"}],"kind":"symbol","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSResponseError","fragments":[{"text":"struct","kind":"keyword"},{"kind":"text","text":" "},{"text":"AWSResponseError","kind":"identifier"}]},"doc://codes.soto.soto-core/documentation/SotoCore/AWSErrorContext":{"url":"\/documentation\/sotocore\/awserrorcontext","title":"AWSErrorContext","navigatorTitle":[{"kind":"identifier","text":"AWSErrorContext"}],"type":"topic","abstract":[{"text":"Additional information about error","type":"text"}],"role":"symbol","kind":"symbol","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSErrorContext","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"text":"AWSErrorContext","kind":"identifier"}]},"doc://codes.soto.soto-core/documentation/SotoCore":{"type":"topic","title":"SotoCore","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore","url":"\/documentation\/sotocore","role":"collection","kind":"symbol","abstract":[{"text":"The core framework for Soto, a Swift SDK for AWS","type":"text"}]}}}